src/main/kotlin/racingcar/Car.kt

```diff
@@ -1,10 +1,36 @@
 package racingcar
 
-class Car(var status: Int = 0) {
-    fun move(condition: Int) {
+class Car(
+    private val _name: String,
+    private var _status: Int = 0,
+) {
+    val name: String
+        get() {
+            return this._name
+        }
```

Comment:
```suggestion
    val name: String = _name
```

커스텀 getter를 사용하지 않고 바로 할당이 가능합니다 😊